impeller/renderer/renderer_dart_unittests.cc

```diff
@@ -33,7 +36,12 @@ class RendererDartTest : public PlaygroundTest,
       : settings_(CreateSettingsForFixture()),
         vm_ref_(flutter::DartVMRef::Create(settings_)) {
     fml::MessageLoop::EnsureInitializedForCurrentThread();
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
     current_task_runner_ = fml::MessageLoop::GetCurrent().GetTaskRunner();
```

Comment:
`fml::MessageLoop::GetCurrent` is an API only meant to be used on the embedder and we can't know if the call is made on the embedder unless you clarify. You can just define FML_USED_ON_EMBEDDER at the very top of the file and the deprecated declaration warning will go away :)

Reply:
Ah, makes sense. Done.